### PR TITLE
Issue #475 Remove some easy warnings

### PR DESCRIPTION
**Pull request by Didou09**
Motivations:
--------------
* The unit tests in [GA](https://github.com/ToFuProject/tofu/runs/2004133116?check_suite_focus=true) showed a few warnings that seemed easy to remove

Main changes:
----------------
* A few minor changes (in particular several occurences of `np.int`  or `np.long` replaced by `int` as recommended [here](https://numpy.org/devdocs/release/1.20.0-notes.html)).

Issues:
-------
* Fixes, in devel, issue #475

### CODE REVIEW

File: tofu/tests/tests01_geom/test_01_GG.py [hunk 1]
```diff
@@ -1797,16 +1797,16 @@ def test24_is_visible(debug=2):
             shot=0
         )
         s1 = tfg.PFC(Name="S1",
-                     Poly=[SP0x, SP0y],
+                     Poly=[SP0x[:-1], SP0y[:-1]],
```
Comment (lasofivec) [inline tofu/tests/tests01_geom/test_01_GG.py:1800]: why these changes ? is it to leave the polygon open ?

**Review by lasofivec — APPROVED** [review]
Ok

File: tofu/tests/tests01_geom/test_01_GG.py (hunk 1 above)
Comment (Didou09) [reply tofu/tests/tests01_geom/test_01_GG.py:1800]: Yes, because the PFC class checks for doubles in the polygons.
If it provided as a closed polygon, it will work, but the class will throw a warning to the user just saying that it detected the double point.
It is a relatively normal behavior (double-checking inputs and keeping the user informed), but for unit tests we already know it and we just want to get rid of easy warnings so really problematic ones are more visible.
